Source/Core/Core/HW/DSP.cpp

```diff
@@ -212,6 +212,7 @@ void EnableInstantDMA()
 	CoreTiming::RemoveEvent(et_CompleteARAM);
 	CompleteARAM(0, 0);
 	instant_dma = true;
+	ERROR_LOG(DSPINTERFACE, "Enabling Instant ARAM DMA hack");
```

Comment:
These kinds of things are actually super useful for me when I'm testing; if I startup a game that uses this, I can see it right away instead of just guessing.

I honestly wish we had more diagnostic data like this.  Can it be a notice or something that's not an error?  It's useful.